app/controllers/stories/tagged_articles_controller.rb

```diff
@@ -59,7 +59,9 @@ def tagged_count(tag:)
     end
 
     def not_found_if_not_established(stories:, tag:)
-      not_found if stories.none? && !tag.supported?
+      return if tag.supported?
+
+      not_found unless stories.any? { |s| s.published? }
```

Comment:
This line gets flagged by Rubocop, so ideally we fix it before it hits `main`.

```suggestion
      not_found unless stories.any?(&:published?)
```